pages/ocf-host-application.js

```diff
@@ -35,6 +35,20 @@ const collectiveQuery = gqlV2/* GraphQL */ `
   }
 `;
 
+const hostQuery = gqlV2/* GraphQL */ `
+  query HostData {
```

Comment:
should be consistent with the variable name and try to be unique, explaining the context

Reply:
or combine with the other query.